src/components/cards-grid-list/components/tutorial-cards-grid-list/index.tsx

```diff
@@ -24,7 +24,7 @@ const TutorialCardsGridList = ({ tutorials, ...restProps }) => {
 		tutorialIds.push(tutorial.id)
 		cardsGridListItems.push(
 			<li key={tutorial.id}>
-				<TutorialCardWithBookmark {...tutorial} />
+				<TutorialCardWithAuthElements {...tutorial} />
```

Comment:
I've modified `TutorialCardWithBookmark` to include progress as well, and renamed to `TutorialCardWithAuthElements`.